Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+//Floyd-Warshall Algorithm is an algorithm for finding the shortest path between all the pairs of vertices in a weighted graph.
+
+$f = fopen('php://stdin', 'r');
+
+$line = fgets($f);
+
+$arr = explode(" ", $line);
+
+$m = $arr[0]; // first int as m rows 
+$n = $arr[1]; // second int as n cols
+
+unset($arr[0]);
+unset($arr[1]);
+
+$arr = array_values($arr);
+$graph = [];
+
+for ($i = 0; $i < $m; $i++)
+{
+    for ($j = 0; $j < $n; $j++) 
+    {
+        $result[$i][$j] = current($arr);
+        next($arr);
+    }
+}
+
+fclose($f);
+
+$INF = 99999;
+
+function PrintResult($distance, $verticesCount)
+{
+    global $INF;	
+    echo "<pre>" . "Shortest distances between every pair of vertices:" . "<br/>";
+
+    for ($i = 0; $i < $verticesCount; ++$i)
+    {
+         for ($j = 0; $j < $verticesCount; ++$j)
+	 {
+             if ($distance[$i][$j] == $INF)
+             {
+                 echo str_pad("INF", 7);
+             }
+             else
+             {
+                echo str_pad($distance[$i][$j], 7);
+             }
+	}
+	echo "<br/>";
```

Comment:
please correct the indentation in curly braces and echo statement

Reply:
@asha15  Done. Please review.